cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -170,16 +170,16 @@ public void testWorkflowNodeState() throws Exception {
     ProgramRunId workflowRun = appId.workflow(workflowName).run(workflowRunId);
 
     // start Workflow
-    store.setStart(workflowRun.getParent(), workflowRun.getRun(), currentTime);
+    store.setStartAndRun(workflowRun.getParent(), workflowRun.getRun(), currentTime, currentTime + 1);
```

Comment:
Well, different invocations of `setStartAndRun` have different delay times - I don't think it makes sense to have a different variable for each one.

Also, we specify the end time for `setStop` as well relative to the startTime (startTime + 50, for example).